packages/analytics-node/src/node-client.ts

```diff
@@ -13,6 +13,10 @@ export class AmplitudeNode extends AmplitudeCore<NodeConfig> {
 
     await this.add(new Context());
     await this.add(new Destination());
+
+    // Flush existing events, which might be collected by track before init
+    // This flush needs to run after plugin installation to gain the correct attributes
+    await this.timeline.flush();
```

Comment:
Node might not need this because it is at backend and events should not be collected before init. Correct me if I miss any scenario.

Reply:
Thanks for reviewing! I think node and react-native might have the similar issue too. As we don't have a hard control on the order of `#init` and `#track`.